contrib/http_util/src/main/java/io/opencensus/contrib/http/util/HttpStatsUtil.java

```diff
@@ -0,0 +1,435 @@
+ * libraries/frameworks to reference and use.
+ *
+ * @since 0.13
+ */
+public class HttpStatsUtil {
+
+  private HttpStatsUtil() {}
+
+  private static final String UNIT_COUNT = "1";
+  private static final String UNIT_SIZE = "By";
+  private static final String UNIT_LATENCY = "ms";
+
+  @VisibleForTesting
+  static String getName(String clientOrServer, String shortName) {
+    return String.format("opencensus.io/http/%s/%s", clientOrServer, shortName);
+  }
+
+  @VisibleForTesting static final Aggregation COUNT = Count.create();
+
+  @VisibleForTesting
+  static final Aggregation SIZE_DISTRIBUTION =
+      Distribution.create(
+          BucketBoundaries.create(
+              Collections.<Double>unmodifiableList(
+                  Arrays.<Double>asList(
+                      0.0,
+                      1024.0,
+                      2048.0,
+                      4096.0,
+                      16384.0,
+                      65536.0,
+                      262144.0,
+                      1048576.0,
+                      4194304.0,
+                      16777216.0,
+                      67108864.0,
+                      268435456.0,
+                      1073741824.0,
+                      4294967296.0))));
+
+  @VisibleForTesting
+  static final Aggregation LATENCY_DISTRIBUTION =
+      Distribution.create(
+          BucketBoundaries.create(
+              Collections.<Double>unmodifiableList(
+                  Arrays.<Double>asList(
+                      0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 8.0, 10.0, 13.0, 16.0, 20.0, 25.0, 30.0,
+                      40.0, 50.0, 65.0, 80.0, 100.0, 130.0, 160.0, 200.0, 250.0, 300.0, 400.0,
+                      500.0, 650.0, 800.0, 1000.0, 2000.0, 5000.0, 10000.0, 20000.0, 50000.0,
+                      100000.0))));
+
+  /**
+   * {@link Measure} for the number of client-side HTTP requests started.
+   *
+   * @since 0.13
+   */
+  public static final MeasureLong HTTP_CLIENT_REQUEST_COUNT =
+      Measure.MeasureLong.create(
+          getName("client", "request_count"),
+          "Number of client-side HTTP requests started",
+          UNIT_COUNT);
+
+  /**
+   * {@link Measure} for the client-side HTTP request body size if set as ContentLength
+   * (uncompressed).
+   *
+   * @since 0.13
+   */
+  public static final MeasureLong HTTP_CLIENT_REQUEST_BYTES =
+      Measure.MeasureLong.create(
+          getName("client", "request_bytes"),
+          "Client-side HTTP request body size if set as ContentLength (uncompressed)",
+          UNIT_SIZE);
+
+  /**
+   * {@link Measure} for the client-side HTTP response body size (uncompressed).
+   *
+   * @since 0.13
+   */
+  public static final MeasureLong HTTP_CLIENT_RESPONSE_BYTES =
+      Measure.MeasureLong.create(
+          getName("client", "response_bytes"),
+          "Client-side HTTP response body size (uncompressed)",
+          UNIT_SIZE);
+
+  /**
+   * {@link Measure} for the client-side end-to-end latency.
```

Comment:
nit: s/end-to-end/roundtrip? (consistent with gRPC metric)

Reply:
SG to me.